content/blog/tina-style-open-authoring.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Open Authoring — Tina-Style
+date: '2020-03-05T07:00:00.000Z'
+author: Scott Gallant
+draft: false
+consumes:
+  - file: null
+    details: TODO - add consumer data
+---
+
+We're focused on improving the independent web as a whole. We want to craft tools that help people build better sites and create better content.
+
+In-line with this mission, we built **Tina-Style Open Authoring** ✍️. [Netlify CMS](https://www.netlifycms.org/) spearheaded the [core idea](https://css-tricks.com/netlify-cms-open-authoring/#article-header-id-0) this past summer. It involves opening up your website to accept content-related _contributions from anyone_.
+
+## Picture this:
+
+On your website, there's an _"Edit this page on GitHub"_ link — familiar enough. But imagine that when you click it, instead of kicking you over to GitHub, the site itself becomes editable — **like a Google Doc**.
+
+The scaffolding underneath Open Authoring is a Git-based workflow 😍. Triggering 'Edit Mode' creates a new fork to track and commit changes on. When content updates are ready, the editor can make a pull-request. The site owner can then review the work before making it live.
```

Comment:
```suggestion
The scaffolding underneath Open Authoring is a Git-based workflow. Triggering 'Edit Mode' creates a new fork to track and commit changes on. When content updates are ready, the editor can make a pull-request. The site owner can then review the work before making it live.
```
Maybe Nitpicky, but at this point, I might not understand why this is heart-worthy until I read more

Reply:
lolz